views/partials/social_media_sharing_buttons.hbs

```diff
@@ -1,13 +1,12 @@
 <ul class="social-media-sharing-buttons">
   <li>
-    <a class="twitter-share-button" href="https://twitter.com/intent/tweet?text=Find%20out%20if%20your%20email%20addresses%20have%20been%20hacked%20at%20https://monitor.firefox.com.">
-      <img src="/img/social_media_icons/twitter.svg" />
-    </a>
-  </li>
-  <li>
-   <img src="/img/social_media_icons/facebook.svg" />
+    <img class="facebook-button" alt="Share Firefox Monitor on Facebook" src="/img/social_media_icons/social-facebook.svg" />
+    Share
   </li>
   <li>
-    <a href="mailto:?&subject=Firefox%20Monitor%20%3A%20Protect%20Your%20Data&body=Firefox%20Monitor%20%3A%20Get%20Michelle%20to%20whip%20up%20a%20blurb%20about%20Firefox%20Monitor%20that%20goes%20here%3F%20%0A%0Ahttps%3A%2F%2Fmonitor.firefox.com/" target="_blank" rel="noopener noreferrer" aria-label="Email"><img src="/img/social_media_icons/mail.svg" /></a>
+    <a class="twitter-share-button" href="https://twitter.com/intent/tweet?text=Find%20out%20if%20your%20email%20addresses%20have%20been%20hacked%20at%20https://monitor.firefox.com.">
+      <img alt="Share Firefox Monitor on Twitter" src="/img/social_media_icons/social-twitter.svg" />
```

Comment:
Why does the Facebook image have a `class="facebook-button"` but the Twitter image doesnt need a custom class?

Reply:
Because of it's shape and proximity to the Twitter logo. It required especial margining to appear evenly spaced from the 'Share.'

Reply:
So are you saying the design will break if we add another logo or reorder the logos?

Should we just add some margining to the "Share" text instead?

Reply:
No it should look totally fine to add icons + text to either side. The images in that ul are set to margin-right: 8px, with a special case being made for the facebook logo since it's much narrower. We could instead add the spacing to the words ('Share', 'Tweet') but we'd have to wrap them in a <span> tag or something first since they're in <li> or <li><a> tags alongside the icons.